sesclient.ps1

```diff
@@ -2,7 +2,17 @@
 # Convenience Launcher for sesclient
 #
 
-$sesclient = "$PSScriptRoot\x64\Debug\sesclient.native.exe"
+# Find the latest version of sesclient.native.exe that was compiled
+# [This will automatically find the right version, even if someone does a release 
+# build or if the output path changes]
+$sesclient = get-childitem $PSScriptRoot\x64\sesclient.native.exe -recurse | sort-object LastWriteTimeUtc | select-object -last 1
+
+if ($sesclient -eq $null) 
+{
+    Write-Host "Could not find 'sesclient.native.exe' within $PSScriptRoot\x64" -ForegroundColor Red
+    Write-Host "Do you need to run a build?" -ForegroundColor Blue
```

Comment:
Blue is a bit hard to read against the default blue Powershell background.

Reply:
😄 I'll tweak it.